mlflow/tracking/client.py

```diff
@@ -73,7 +76,7 @@ def create_run(self, experiment_id, user_id=None, run_name=None, start_time=None
         # Extract run attributes from tags
         # This logic is temporary; by the 1.0 release, this information will only be stored in tags
         # and will not be available as attributes of the run
-        parent_run_id = tags.get(MLFLOW_PARENT_RUN_ID)
+        parent_run_id = parent_run_id or tags.get(MLFLOW_PARENT_RUN_ID)
```

Comment:
what if `parent_run_id` is `""` should we silently accept whatever `tags.get(MLFLOW_PARENT_RUN_ID)`

Reply:
Good catch - it would be confusing to fall through to `tags.get(MLFLOW_PARENT_RUN_ID)` in that case.

To address the comment I changed this to explicitly check whether the parent_run_id argument is `None` and only use the value from `tags` if that's the case (that is, we only use `tags.get(MLFLOW_PARENT_RUN_ID)` if `parent_run_id` is unspecified, or if the user for some reason passes `parent_run_id=None